### PR TITLE
Landing page chart one

### PR DESCRIPTION
**Pull request by JosephDClark97**
Styled the first Bit Coin Landing Page Chart and exported those components

### CODE REVIEW

File: src/components/BitChart/BitChart.styles.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+
+export const BitContainer = styled.div`
+display:flex;
+height:449px;
+width:833px;
+background-color:#191B1F;
+border-radius:1px solid #707070;
```
Comment (yoloOnTheBattlefield) [inline src/components/BitChart/BitChart.styles.js:8]: indent

File: src/components/CoinTable/CoinTable.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    async componentDidMount() {
+     await fetch('https://api.coingecko.com/api/v3')
+        .then(result => {
+          this.setState(
+            {
+              result: [...result.content]
+            }
+            )
+        })
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:18]: use async await

File: src/components/CoinTable/CoinTable.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      render() {
+        return ( 
+        <div> 
+          {
+          this.state.list.map
+          (
+            el => 
+            <div>
+            {el.name}
+            </div>
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinTable/CoinTable.js:30]: indent

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/BitChart/BitChart.styles.js (hunk 1 above)
Comment (JosephDClark97) [reply src/components/BitChart/BitChart.styles.js:8]: completed

File: src/components/CoinTable/CoinTable.js (hunk 2 above)
Comment (JosephDClark97) [reply src/components/CoinTable/CoinTable.js:18]: how should i use it with this line ? 
what does it look like syntaxically?

**Pull request closed without merging**